README.md

```diff
@@ -1,17 +1,19 @@
-# Capstone-react
+# Capstone-react | COVID19 STATS
 
 ## Description
 
 A covid19 stats app build as a ReactJS Capstone project at Microverse. Built with React, Redux and API.
 
 ## Screenshot
 
-![image: coming soon](https://paceholder.png)
+![Capstone React](https://user-images.githubusercontent.com/27709832/157883026-89e3ecca-f26d-46c6-9917-2dc1dc072316.png)
 `Current screenshot`
 
 ## Live Demo
 
-View the [Coming Soon...](https://placeholder.jpg) here ☕
+View the [live Demo page](https://capstone-react-brytebee.herokuapp.com/) here ☕
+
+[Developer's Description](https://drive.google.com/file/d/18rkjca1hhUgLte_w8rFbBn1GGae5gasu/view?usp=sharing)
```

Comment:
Thanks bro! 😄

Reply:
@brytebee You are welcome. This can happen 1 time from each 100 review but it did with you 🙌. I mean review same code the second time. Now its time. **Good luck 🍀☕**